api/src/services/auth/group/group.schema.ts

```diff
@@ -0,0 +1,131 @@
+import { Type, querySyntax, Static, getDataValidator, getValidator } from '@feathersjs/typebox'
+import { dataValidator, queryValidator } from '../../../commons/validators'
+import { workspaceSchema } from '../../workspace/workspace.schema'
+import { workspaceOwnerSchema } from '../user/user.schema'
+
+// Schema for the basic data model (e.g. creating new entries)
+export const groupSchema = Type.Object(
+  {
+    id: Type.String({ format: 'uuid' }),
+    name: Type.String({
+      description: 'Name of the group',
+    }),
+    documentation: Type.Optional(
+      Type.String({
+        description: 'Documentation',
+      }),
+    ),
+    workspaceId: Type.String({
+      format: 'uuid',
+      description: 'Related workspace',
+    }),
+    roleId: Type.String({
+      format: 'uuid',
+      description: 'Related role of the workspace',
+    }),
+
+    workspace: Type.Optional(
```

Comment:
What happened if a group is related to several workspaces ? It is possible ?

Reply:
Actually, no...
Do you think it's a usecase ?

Reply:
Not really :)